src/Admin/BaseMediaAdmin.php

```diff
@@ -90,7 +103,8 @@ public function configurePersistentParameters(): array
         $categoryId = $this->getRequest()->get('category');
 
         if (null !== $this->categoryManager && !$categoryId) {
-            $category = $this->categoryManager->getRootCategory($context);
+            $categories = $this->categoryManager->getRootCategoriesForContext($this->contextManager->find($context));
+            $category = $categories[0];
```

Comment:
I don't like a lot having to do this kind of access to the array, but previously this returned one root category, now returns all (whatever that means). Same for the other places.

Reply:
Category are set this way
```
$this->categories[$context->getId()][$category->getId()] = $category;
```
So I'm not sure `[0]` is the right way to do it.
`current($categories)` maybe instead ?

Reply:
Updated.